prow/hook/events.go

```diff
@@ -24,6 +24,32 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
+const failedCommentCoerceFmt = "Could not coerce %s event to a GenericCommentEvent. Unknown 'action': %q."
+
+var (
+	nonCommentIssueActions = []github.IssueEventAction{
```

Comment:
maybe use a map / sets.String for these?

Reply:
Done.